state/state_test.go

```diff
@@ -552,6 +552,9 @@ func (helm *mockHelmExec) DeleteRelease(name string, flags ...string) error {
 func (helm *mockHelmExec) DecryptSecret(name string) (string, error) {
 	return "", nil
 }
+func (helm *mockHelmExec) TestRelease(name string, flags ...string) error {
+	return nil
```

Comment:
If I could ask for more - Would you mind adding a test for `TestReleases`?

Probably you'd make this func look like the sub [ReleaseStatus](https://github.com/roboll/helmfile/pull/150/files#diff-a1153b96e079da8da43231ad48de57a0R542) func:

```golang
func (helm *mockHelmExec) TestRelease(name string, flags ...string) error {
 	if strings.Contains(name, "error") {
 		return errors.New("error")
 	}
 	helm.releases = append(helm.releases, mockRelease{name: name, flags: flags})
 	return nil
 }
```

And write a test involves the stub like https://github.com/roboll/helmfile/pull/150/files#diff-a1153b96e079da8da43231ad48de57a0R700